### DISCUSSION TITLE
White page after click on install of YOURLS 1.9.2

### DISCUSSION THREAD

**Discussion by BarnyGumble** [YOURLS/YOURLS#3813] [Q&A] [answered]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/YOURLS/.github/blob/main/CODE_OF_CONDUCT.md).

### Submission validity

- [X] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions).
- [X] I checked [current issues](https://github.com/YOURLS/YOURLS/issues) and this request isn't a duplicate of an existing issue, opened or closed.

### Self troubleshooting

- [X] I've read the [documentation](https://yourls.org/) and made sure to follow it.
- [X] I've read the [troubleshooting first steps](https://docs.yourls.org/guide/troubleshooting/first-steps.html) and [frequent issues](https://docs.yourls.org/guide/troubleshooting/frequent-issues.html) guides.

### Version

1.9.2

### Description

After clean install of version 1.9.2 I have a white page when I click on install. After upgrade is the same problem.
Plugins are deactivated.
Version before I have was 1.7.2. This works whithout problems with PHP 7.4.
Current PHP Version is 8.2.
In the .httaccess I change the version to 8.2.
I activate the debug function, but I don't have any error. The database can be load.
Please help me with this issue.

### Expectation

I expected the green installation steps to begin on admin-backend. 

### Reproduction steps

go to https://mydomain.com/admin/

there open the page of to https://mydomain.com/admin/install.php

after click on install there is a white page
ttps://mydomain.com/admin/install.php?

### Context

I use a hosting server at Strato Germany with a database which I have since 2013.
before with version 1.7.2 and PHP 7.4
now I need to use PHP 8.2.
I change the version on Strato homepage to PHP 8.2 and in the .htaccess

**Comment by dgw** [comment]
If YOURLS fails to generate a page, debug output won't help much. You'll need to [look at server logs](https://yourls.org/docs/guide/troubleshooting/first-steps#check-server-logs).

I couldn't muddle my way through Google Translate well enough to get useful results out of Strato's help center in German, but I'm sure PHP errors are logged somewhere. If you can find the right file, you'll see from the error traceback exactly where/why YOURLS errors out and probably get a good idea of how to fix it.

**Reply by BarnyGumble** [reply]
This is the server log, which I can see:

[21/Sep/2024:01:08:19 +0200] "HEAD /admin/install.php HTTP/1.1" 200 - "https://www.******.de/wp" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36"
*****.de anon-0-0-0-201.ip6.invalid -

**Reply by dgw** [reply]
That is a quite old Chrome user-agent. And a HEAD request naturally won't fetch any page body. Do you actually have YOURLS installed under `/wp`? Because if not, you need to find the actual log for where YOURLS is hosted.

**Reply by BarnyGumble** [reply]
I don't know.
I have installed YOURLS on a separate hosting of german Strato host and not at WP.
The server log I send is from the log of Strato. I don't know, where I can find any more logs.
I have all my 4 domains at Strato. 3 of them are connected with WP Database.
YOURLS is on a separate Database with another domain. On this domain was a forwarding to the main domain before a had installed WP on the other domains.

I have delete all folders and datas from YOURLS on this database and install it all new, but this dosn't resolve the problem with the white screen.
The old version 1.7.2 runs without any problems.

**Reply by dgw** [reply]
It's not really within my ability, as someone with no access to or knowledge of the server, to divine where you might find PHP's error logs. The best I was able to come up with by searching Strato's help center was this page: https://www.strato.de/faq/hosting/wo-finde-ich-meine-logfiles/#bereitstellung_errorlogs

Maybe it will give you a starting point to find more information.

One _wild guess_ would be your installed PHP version; YOURLS 1.9 requires PHP 7.4 or higher. If you are still running PHP 5, or an older version of PHP 7, newer YOURLS releases won't be able to run.

**Reply by BarnyGumble** [reply]
This is the error-log-file of my Strato-Domain:

25.09.2024 16:08:28 domain.de [client 37.60.127.0] ModSecurity: Warning. Match of "rx (^/file\\\\?file=/etc/cccam\\\\.cfg$|event=update_asl_config|^/etc/js/|^/index\\\\.php\\\\?module=asl&event=)" against "REQUEST_URI" required. [file "/opt/RZapache/conf/rules/10_asl_rules.conf"] [line "234"] [id "390709"] [rev "25"] [msg "Atomicorp.com UNSUPPORTED DELAYED Rules: Attempt to access protected file remotely"] [data "../etc/"] [severity "CRITICAL"] [hostname "domain.de"] [uri "/maint/modules/home/index.php"] [unique_id "ZvQZXIKkzC7fruSesWwiGAAAM04"]

I use the version PHP 8.2 for YOURLS 1.9.
This is the .htaccess-file:

```
AddType application/x-httpd-php82 .php

# BEGIN YOURLS
<IfModule mod_rewrite.c>
RewriteEngine On
RewriteBase /
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^.*$ /yourls-loader.php [L]
</IfModule>
# END YOURLS


# BEGIN WordPress
<IfModule mod_rewrite.c>
RewriteEngine On
RewriteBase /
RewriteRule ^index\.php$ - [L]
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule . /index.php [L]
</IfModule>

# END WordPress
```

**Reply by BarnyGumble** [reply]
How can I install YOURLS completely new?
I have delete all files and folders of YOURLS of my FTP Server and copy all files of YOURLS again on the same place.
But this dosn't changes anything. The Problem with the white screen is the same.

When I change all files from my backup of YOURLS Version 1.7.2 with PHP 7.4 all is working correctly.
After copy of all files and folders of version 1.9.2 and change to PHP version 8.2 I get a white page, after click on install at the Admin-Page.
I test it with PHP version 7.4 with the same version 1.9.2 of YOURLS. But I it is the same problem.

The same problem with YOURLS version 1.7.3 with PHP version 7.4. (White screen - impossible to install)
The last YOURLS version what working without problems is 1.7.2 with PHP version 7.4.

**Reply by BarnyGumble** [reply]
> This is the error-log-file of my Strato-Domain:
> 
> 25.09.2024 16:08:28 domain.de [client 37.60.127.0] ModSecurity: Warning. Match of "rx (^/file\\?file=/etc/cccam\\.cfg$|event=update_asl_config|^/etc/js/|^/index\\.php\\?module=asl&event=)" against "REQUEST_URI" required. [file "/opt/RZapache/conf/rules/10_asl_rules.conf"] [line "234"] [id "390709"] [rev "25"] [msg "Atomicorp.com UNSUPPORTED DELAYED Rules: Attempt to access protected file remotely"] [data "../etc/"] [severity "CRITICAL"] [hostname "domain.de"] [uri "/maint/modules/home/index.php"] [unique_id "ZvQZXIKkzC7fruSesWwiGAAAM04"]
> 
> I use the version PHP 8.2 for YOURLS 1.9. This is the .htaccess-file:
> 
> ```
> AddType application/x-httpd-php82 .php
> 
> # BEGIN YOURLS
> <IfModule mod_rewrite.c>
> RewriteEngine On
> RewriteBase /
> RewriteCond %{REQUEST_FILENAME} !-f
> RewriteCond %{REQUEST_FILENAME} !-d
> RewriteRule ^.*$ /yourls-loader.php [L]
> </IfModule>
> # END YOURLS
> 
> 
> # BEGIN WordPress
> <IfModule mod_rewrite.c>
> RewriteEngine On
> RewriteBase /
> RewriteRule ^index\.php$ - [L]
> RewriteCond %{REQUEST_FILENAME} !-f
> RewriteCond %{REQUEST_FILENAME} !-d
> RewriteRule . /index.php [L]
> </IfModule>
> 
> # END WordPress
> ```

When I edit in the .htaccess-file the version of PHP from 7.4 e.g. to 8.0, YOURLS produced a white screen.
Similar it don't affect when I only edit the PHP version on strato-website and not in the .htaccess-file.
When I edit both, there is a white screen, too.
This will be do at version 1.7.2 and all laters, too.
I can imagine that there is somewhere the probleme with the .htaccess-file or something with the PHP-version. But I don't know what's wrong.

**Reply by jpowell53590** [reply, marked as answer]
I would not mix WordPress with YOURLs in the same folder.

Instead, I would recommend setting up a subdomain like go.domain.de and on the web server point go.domain.de to the public_html/yourls folder
